### DISCUSSION TITLE
Dịch Stick-Breaking Process

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#65] [Terminology] [answered]
Trong chương Unsupervised Learning có thuật ngữ Stick-Breaking Process. Quy trình này được hiểu nôm na là chia một đoạn có độ dài `l` ra làm 2, lần chia đầu tiên tại X, phân phối đều trong đoạn `[0,l]`, tiếp tục chia đoạn X làm đôi tại Y, phân phối đều trong đoạn [0,X], v.v.
Video này giải thích rõ quy trình và ứng dụng của nó: https://www.youtube.com/watch?v=aXFbBcabaQA

Mình đề xuất dịch là `Quá trình Chia Đoạn`.

Mọi người nghĩ thế nào?

@mlbvn/handson-ml2

**Comment by minhduc0711** [comment, marked as answer]
Chốt là `Quá trình Chia Đoạn` nhé.
